oci-build-examples/oci-build-caching/SETUP-PROJECT.md

```diff
@@ -0,0 +1,72 @@
+Hello World sample shows how to deploy [SpringBoot](http://projects.spring.io/spring-boot/) RESTful web service application with [Docker](https://www.docker.com/)
+
```

Comment:
What is the usage of this file ? Especially with docker compose

Reply:
This file is to bring up the service in the local machine for development or modification.